README.md

```diff
@@ -12,6 +12,12 @@ To suggest an improvement, send a pull request or [open an issue](https://github
 
 
 - [Basics](#basics)
+- [Threat Modeling](#threat-modeling)
+   * [Step 1](#step-1-identify-valuable-assets)
```

Comment:
Remove the steps and just use the section titles.

Reply:
done